package.json

```diff
@@ -28,7 +28,7 @@
     "lint": "eslint",
```

Comment:
@claireclark1 Here's the issue. The dependencies of tailwind increased the minimum needed to `es2015` (which is fine to use) so this line:
`"build:tsc": "tsc tailwind.config.ts src/jsx.d.ts --outDir dist --declaration",`
Needs to be:
`"build:tsc": "tsc --lib es2015 tailwind.config.ts src/jsx.d.ts --outDir dist --declaration",`

Then it will all work.